doc/coq.rst

```diff
@@ -235,6 +242,7 @@ The supported Coq language versions (not the version of Coq) are:
 - ``0.3``: Support for ``(mode native)`` requires Coq >= 8.10 (and Dune >= 2.9
   for Coq >= 8.14).
 - ``0.4``: Support for interproject composition of theories.
+- ``0.6``: Support for ``(stdlib no)``.
```

Comment:
Let's include the change from 0.5 here so that the doc doesn't look incomplete.

```suggestion
- ``0.5``: ``(libraries ...)`` field deprecated in favor of ``(plugins ...)`` field.
- ``0.6``: Support for ``(stdlib no)``.
```